dsaas-services/f8-env.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: fead97763ad02a318ccce057cf84c0c1c1681480
+  name: fabric8-env
+  path: /openshift/f8env.app.yaml
+  url: https://github.com/fabric8-services/fabric8-env/
+  hash_length: 6
```

Comment:
I see, some has hash_length 6 and some has 7.  What it should be and where it's used ??